spec/presenters/root_browse_page_presenter_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe RootBrowsePagePresenter do
+
+  describe "#render_for_publishing_api" do
+    it "is valid against the schema without browse pages", :schema_test => true do
+      rendered = RootBrowsePagePresenter.new.render_for_publishing_api
+
+      expect(rendered).to be_valid_against_schema('mainstream_browse_page')
+    end
+
+    it "renders valid top level browse pages" do
+      top_level_page_1 = create(:mainstream_browse_page)
+      top_level_page_2 = create(:mainstream_browse_page)
+
+      rendered = RootBrowsePagePresenter.new.render_for_publishing_api
+
+      expect(rendered).to be_valid_against_schema('mainstream_browse_page')
+      expect(rendered[:links]["top_level_browse_pages"]).to eq([
+        top_level_page_1.content_id,
+        top_level_page_2.content_id,
+      ])
```

Comment:
These 2 assertions could be split into separate tests wrapped in a context block.  The schema one could then be tagged as a schema test so that it gets included in that test run.